src/explore-education-statistics-admin/src/components/EditReleaseSetupSummary.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Release } from '@admin/services/publicationService';
+import { format } from 'date-fns';
+import Link from '@admin/components/Link';
+
+interface Props {
+  publicationTitle: string;
+  release: Release;
+}
+
+const EditReleaseSetupSummary = ({ publicationTitle, release }: Props) => {
```

Comment:
We should try to keep the contents of the top-level `src/components` as generic as possible. Keep page-specific components as close to their relevant page(s) as possible, so in this case, this component can go under a directory like `src/pages/edit-release/components`.

Reply:
Cool, didn't know there was that convention, but that'll make it much neater.

Reply:
Done - got an edit-release folder going awn with all individual pages and page templates in now